posts/how-to-integrate-aws-connect.md

```diff
@@ -0,0 +1,177 @@
+---
+authors:
+- John Strickler
+tags:
+- AWS Connect
+- Agent Servicing
+- Javascript
+date: 2022-04-29T00:00:00.000Z
+title: "How To Integrate AWS Connect"
+image: 
+---
+
+In this post, I demonstrate how to easily integrate [AWS Connect](https://aws.amazon.com/connect) ("Connect") to any new or existing web application. Connect is AWS's managed solution for enabling omnichannel communication capabilities to your organization. It provides the ability to fully handle human interactions from a single, streamling user interface including features such as receiving inbound calls, making outbound calls, and having real-time chats.
+
+If you're at all intimiated by this, don't be. AWS has done a fantastic job in making Connect easy to get started with by providing a step-by-step setup process with a healthy set of defaults. They built it to be highly configurable and developer friendly by providing Connect-specific SDKs, APIs, and a robust administrator portal that enables no-code workflows and drag-and-drop configuration.
+
+If you're experimenting then you'll be glad to hear that Connect is covered by the [AWS free tier](https://aws.amazon.com/connect/pricing#AWS_Free_Tier). There's a good amount of free coverage for Connect services. You shouldn't receive any charges by using a baseline setup, but as always, keep a close watch on usage so you can manage any incurred costs.
+
+## Creating and configuring the AWS Connect instance
+
+The first step is to create an AWS Connect instance. Log in to your AWS account, navigate to the AWS Connect service, and select "Create instance" from the button in the top-right corner. This will bring up a setup wizard. The default settings are good enough to get you started but adjust accordingly based on your needs. Just remember to save the administrator **username** and **password** that you configure because you will be using it later to configure your instance.
```

Comment:
```suggestion
The first step is to create an AWS Connect instance. Log in to your AWS account, navigate to the AWS Connect service, and select "Create instance" from the button in the top right corner. This will bring up a setup wizard. The default settings are enough to get you started, but adjust accordingly based on your needs. Just remember to save the administrator **username** and **password** that you configure because you will be using it later to configure your instance.
```